src/Shouldly.Tests/Strings/ShouldNotMatch.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Shouldly.Tests.TestHelpers;
+
+namespace Shouldly.Tests.Strings
+{
+    public class ShouldNotMatch : ShouldlyShouldTestScenario
+    {
+        protected override void ShouldThrowAWobbly()
+        {
+            "Cheese".ShouldNotMatch(@"\w+", "Some additional context");
+        }
+
+        protected override string ChuckedAWobblyErrorMessage
+        {
+            get
+            {
+                return "\"Cheese\" should not match \"\\w+\" but was \"Cheese\" " +
```

Comment:
Similar to the `ShouldNotBe`, if this test fails the `but was...` will always repeat the expected.

I would just make the error `"\"Cheese\" should not match \"\\w+\" but did`